hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -232,6 +233,8 @@ private FlinkOptions() {
   // ------------------------------------------------------------------------
   //  Write Options
   // ------------------------------------------------------------------------
+  public static final ConfigOption<Integer> SINK_PARALLELISM = FactoryUtil.SINK_PARALLELISM;
+
   public static final ConfigOption<String> TABLE_NAME = ConfigOptions
```

Comment:
Hello, it seems that we want to add a default parallelism to the sink, but there is already parallelism param named `write.tasks`, can we rename this param to `write.default_tasks` or something ? And we should add test in `TestHoodieTableFactory`, before we make agreement, can you revert this PR first ? Thanks ~

Reply:
> Hello, it seems that we want to add a default parallelism to the sink, but there is already parallelism param named `write.tasks`, can we rename this param to `write.default_tasks` or something ? And we should add test in `TestHoodieTableFactory`, before we make agreement, can you revert this PR first ? Thanks ~

well, let me revert this PR first, Then come to an agreement, sorry @hehuiyuan  my mistake.

Reply:
@danny0405 ok.